accepted/PSR-6-cache-meta.md

```diff
@@ -300,7 +300,7 @@ The following sample code is recommended in order to enforce the type check on
 the expiresAt() method:
 
 ```php
-if (! ($expiration instanceof \DateTime || $expiration instanceof \DateTimeImmutable)) {
+if (! (is_null($expiration) || $expiration instanceof \DateTime || $expiration instanceof \DateTimeInterface)) {
```

Comment:
Albeit micro-optimization, `null === $expiration` is much faster as it doesn't require a function call. About unary operators, I believe @michaelcullum is right, but we've used a different style here: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-6-cache-meta.md#41-chosen-approach